llarp/service/sendcontext.cpp

```diff
@@ -57,6 +57,7 @@ namespace llarp
     {
       SharedSecret shared;
       ProtocolFrame f;
+      f.C.Zero();
```

Comment:
Move this to `ProtocolFrame`‘s constructor instead?

Reply:
nvm it's already in it using the `Clear()` function